library/src/scala/runtime/stdLibPatches/language.scala

```diff
@@ -56,6 +61,7 @@ object language:
      *  symbol literals are still supported with a language import, but new software
      *  should not use them.
      */
+    @compileTimeOnly("`symbolLiterals` can only be used at compile time in import statements")
```

Comment:
`object deprecated` above should also be `@compileTimeOnly` since it doesn't exist in the scala 2 scala.language